1.2/test/run

```diff
@@ -157,9 +158,9 @@ test_application() {
 
   test_scl_usage "elixir --version" "OTP 18"
   check_result $?
-  test_scl_usage "elixir --version" "Elixir 1.3.1"
+  test_scl_usage "elixir --version" "Elixir 1.2.5"
```

Comment:
It is only downgraded in the `1.2` folder. There are other directories for version `1.3` and possibly `1.4` in the future. jtslear/s2i-phoenix#1